domains/mubeen.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "iamroming"
+    },
+    "record": {
+        "A": "shopv.in"
```

```suggestion
        "A": "shopv.in"
```
1. A records must be an array of IPs, and this is neither an IP nor an array.